### ISSUE TITLE
Latest release of expect is breaking Redux build

### ISSUE THREAD

**Issue by ellbee** [mjackson/expect#51]
Seems to be a problem with object key ordering.

```js

    // This fails:
    expect(
      [
        {
          type: types.COMPLETE_TODO,
          id: 0
        }, {
          type: types.CLEAR_COMPLETED
        }, {
          type: types.ADD_TODO,
          text: 'Write more tests'
        }
      ].reduce(todos, [
        {
          id: 0,
          completed: false,
          text: 'Use Redux'
        }, {
          id: 1,
          completed: false,
          text: 'Write tests'
        }
      ])
    ).toEqual([
      {
        text: 'Write more tests',
        completed: false,
        id: 2
      }, {
        text: 'Write tests',
        completed: false,
        id: 1
      }
    ])

   // Changing the key order in the argument of `toEqual` as follows passes:
   expect(
      [
        {
          type: types.COMPLETE_TODO,
          id: 0
        }, {
          type: types.CLEAR_COMPLETED
        }, {
          type: types.ADD_TODO,
          text: 'Write more tests'
        }
      ].reduce(todos, [
        {
          id: 0,
          completed: false,
          text: 'Use Redux'
        }, {
          id: 1,
          completed: false,
          text: 'Write tests'
        }
      ])
    ).toEqual([
      {
        id: 2,
        completed: false,
        text: 'Write more tests'
      }, {
        id: 1,
        completed: false,
        text: 'Write tests'
      }
    ])

```

Could be due to [this issue](https://github.com/ljharb/is-equal/issues/3) in the new is-equal library

**Comment by ljharb** [comment]
This is now fixed and released in `is-equal` `v1.3.1`. Sorry about that!

**Comment by ellbee** [comment]
Yep, all good now. Thanks @ljharb.
